src/checks/linkOnGithubAbout.ts

```diff
@@ -0,0 +1,43 @@
+import execa from 'execa';
+import fetch from 'node-fetch';
+import cheerio from 'cheerio';
+
+export const title = 'Github Repository contains link on About Section';
+
+export default async function linkOnGithubAbout() {
+  const { stdout: repoUrlStdout } = await execa.command(
+    'git remote get-url origin',
+  );
+
+  const html = await (await fetch(repoUrlStdout)).text();
+
+  const urlMatch = /[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)?/gi;
+  const regex = new RegExp(urlMatch);
+
+  const $ = cheerio.load(html);
+  const aboutHeading = $('h2')
+    .filter(function(this: Node) {
+      return (
+        $(this)
+          .text()
+          .trim() === 'About'
+      );
+    })
+    .next()
+    .next()
+    .text()
+    .trim();
```

Comment:
This technique is:

- too strict in one way: needs to match that regex
- too lax in another way: only needs to find a URL **somewhere** in the text

I would suggest trying to match only existence of the `.octicon.octicon-link` and the `a[href]` elements - this will be enough to confirm that they've added some link.